CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 1.3.1
+### Added
+* New `environment` option in the `run` closure to pass environment variables to Dev App Server ([#128](../../pulls/128))
+* Automatically read environment from `appengine-web.xml` [appengine-plugins-core/378](https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/378)
```

Comment:
isn't that covered in 128? anyway, yeah I'll update it. More info is always better.